extensions/spring-data-jpa/deployment/src/main/java/io/quarkus/spring/data/deployment/generate/GenerationUtil.java

```diff
@@ -53,4 +60,51 @@ static MethodDescriptor toMethodDescriptor(String generatedClassName, MethodInfo
         return MethodDescriptor.ofMethod(generatedClassName, methodInfo.name(), methodInfo.returnType().name().toString(),
                 parameterTypesStr.toArray(new String[0]));
     }
+
+    static String getSimpleName(DotName ifaceName) {
+        return (ifaceName.isInner() ? ifaceName.local() : ifaceName.withoutPackagePrefix());
+    }
+
+    static DotName createImplDotName(DotName ifaceName) {
```

Comment:
Perhaps this and the other newly added methods should be moved to a different utility class that is more aptly named?

Reply:
This only seems to be used in one class, so could also stay as a private method there.

Reply:
Moved to private methods re: other feedback from @gytis